doc/config.rst

```diff
@@ -294,13 +294,20 @@ General configuration
 
 .. confval:: numfig_format
 
-   A dictionary mapping ``'figure'``, ``'table'`` and ``'code-block'`` to
-   strings that are used for format of figure numbers. Default is to use
-   ``'Fig. %s'`` for ``'figure'``, ``'Table %s'`` for ``'table'`` and
-   ``'Listing %s'`` for ``'code-block'``.
+   A dictionary mapping ``'figure'``, ``'table'``, ``'code-block'`` and
+   ``'section'`` to strings that are used for format of figure numbers.
+   As a special character, `%s` and `{number}` will be replaced to figure
```

Comment:
@tk0miya I don't understand this. I tried it in `numfig_format` but I failed (html). Aren't `{number}`  and `{name}` working only with ``:numref:``, like ``:numref:`Fig. {number}, {name}`  ? 

I wanted to check without the `%s` because I feel the code at https://github.com/sphinx-doc/sphinx/blob/stable/sphinx/writers/latex.py#L558 is missing ``\makeatletter/\makeatother` but I am not sure this branch is ever taken.